client.go

```diff
@@ -178,6 +179,9 @@ func (d processInOption) Value() interface{} { return time.Duration(d) }
 // ErrDuplicateTask error only applies to tasks enqueued with a Unique option.
 var ErrDuplicateTask = errors.New("task already exists")
 
+// ErrEmptyTypename indicates that task's typename is empty.
+var ErrEmptyTypename = errors.New("task typename cannot be empty")
```

Comment:
Like I said in the last comment, I don't think we should export this error value from package API (I don't expect many users encountering this error anyway).